test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BulkDeleteTests.cs

```diff
@@ -177,6 +179,136 @@ public async Task GivenPurgeBulkDeleteRequest_WhenCompleted_ThenHistoricalRecord
             Assert.Equal(resource.VersionId, current.Resource.VersionId);
         }
 
+        [SkippableFact]
+        public async Task GivenBulkDeleteJobWithIncludeSearch_WhenCompleted_ThenIncludedResourcesAreDeleted()
+        {
+            CheckBulkDeleteEnabled();
+
+            var resourceTypes = new Dictionary<string, long>()
```

Comment:
Would it make sense to have multiple links / resource types in this test? Could make the test more robust.

Reply:
Do you mean like having another include statement and a resource for it to find?

Reply:
I was thinking (resource w/ reference) -> (two different resources).

Example is `Encounter.participant[x].actor` or `Encounter.appointment`. 

If this doesn't add value - can be skipped - but my though was > 1 resource would be a more full test.